src/foam/color/colorlib.js

```diff
@@ -242,11 +242,12 @@ foam.LIB({
       var scale = desired/gr;
       return [ scale * r, scale * g, scale * b ];
     },
+
     function randomColor( bounds /* { h/s/l: [min, max] } */ ) {
       let hue = bounds?.h ?? [0, 360];
       let sat = bounds?.s ?? [0, 100];
       let lig = bounds?.l ?? [0, 100];
-      return 'hsl(' + (hue[0] +(hue[1] - hue[0]) * Math.random()) + ',' +
+      return "hsl(" + (hue[0] +(hue[1] - hue[0]) * Math.random()) + ',' +
```

Comment:
Why this change? The coding standard is to use ' unless there's a good reason to use " or `.